hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/TestCleanupTableInfoResponses.java

```diff
@@ -0,0 +1,331 @@
+package org.apache.hadoop.ozone.om.response;
```

Comment:
Please add the Apache License comment to the top of any new files you create in the project.